### PR TITLE
Fixes the dependency injection in compiled js.

### PR DESCRIPTION
**Pull request by ychaker**
# Acceptance criteria
- When `splainer-search` is used in another app that minifies the js, app does not break (eg. Quepid)

### CODE REVIEW

File: services/normalDocSvc.js [hunk 1]
```diff
@@ -5,197 +5,198 @@
...
+      this.createNormalDoc = function(fieldSpec, doc, altExplainJson) {
+        var nDoc = new NormalDoc(fieldSpec, doc);
+        var explJson;
+        if (altExplainJson) {
+          explJson = altExplainJson;
+        } else {
+          explJson = getDocExplain(doc, nDoc);
         }
-        return hotOutOf;
+        return this.snippetDoc(this.explainDoc(nDoc, explJson));
```
Comment (bradfordcp) [inline services/normalDocSvc.js:176]: Is this logic related to the dependency injection changes?

_Note there should be a bit more context here. Evaluate the preceding chunk changes as well._

Comment (ychaker) [inline services/normalDocSvc.js:176]: @bradfordcp i think this is a problem with properly displaying the diff.
the change is actually indenting the code properly (since the encompassing function has been added inside an array, so everything inside the function needed to be pushed)

Comment (bradfordcp) [inline services/normalDocSvc.js:176]: Resolved as rendering bug in Github. There was no logic change, just formatting. Confirmed with @ychaker

**Comment by bradfordcp** [comment]
For record keeping purposes this is in the Angular docs.

screen shot 2015-12-07 at 11 55 30 am
